pages/common/traefik.md

```diff
@@ -0,0 +1,19 @@
+# traefik
+
+> Traefik HTTP reverse proxy and load balance.
```

Comment:
No need to mention traefik again in the heading. Also typo in balance.

I suggest - `A HTTP reverse proxy and load balancer`